src/net/rpc.rs

```diff
@@ -2982,27 +3096,30 @@ impl ConversationHttp {
     }
 
     /// Make a new getinfo request to this endpoint
-    pub fn new_getinfo(&self) -> HttpRequestType {
-        HttpRequestType::GetInfo(HttpRequestMetadata::from_host(self.peer_host.clone()))
+    pub fn new_getinfo(&self, stacks_height: Option<u64>) -> HttpRequestType {
```

Comment:
Why does `getinfo` need a stacks height?  Doesn't the message body _already_ return this information?

Reply:
Also, `/v2/info` isn't used by any of the p2p networking protocols, so it doesn't really help peers here.

Reply:
I added this info just for the test - I do a check in the test for get info that checks that the client and server's ConversationHttp objects get updated appropriately. 

I can test in a different and separate function if this seems confusing.

Reply:
Ah, makes sense